Microsoft.Toolkit.Uwp.UI.Controls/InfiniteCanvas/InfiniteCanvas.cs

```diff
@@ -166,7 +166,7 @@ protected override void OnApplyTemplate()
             _canvasTextBoxTools = (StackPanel)GetTemplateChild("CanvasTextBoxTools");
             _canvasTextBoxColorPicker = (ColorPicker)GetTemplateChild("CanvasTextBoxColorPicker");
             _canvasTextBoxFontSizeTextBox = (TextBox)GetTemplateChild("CanvasTextBoxFontSizeTextBox");
-            _canvasTextBoxItlaicButton = (ToggleButton)GetTemplateChild("CanvasTextBoxItlaicButton");
+            _canvasTextBoxItalicButton = (ToggleButton)GetTemplateChild("CanvasTextBoxItalicButton");
```

Comment:
I was going to say changing the name in the Style is going to be a breaking change, but we don't actually call them out as `TemplatePart`s like we should... (for instance see [TabView](https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/master/Microsoft.Toolkit.Uwp.UI.Controls/TabView/TabView.cs#L20)).

@BurkusCat would you be up for fixing that too?  Let me know if you want to do it as part of this change or another.

Reply:
@michael-hawker I can take a look at that next weekend.

Reply:
@BurkasCat I'm approving this PR, just let me know, but we can merge this one first, and just open another one for those changes if you get to them. Thanks for the contribution! 🎉

Reply:
@BurkusCat Is there any update on this or still being worked on?

Reply:
@Kyaa-dost Apologies, I did not get a chance to look at this. I can try take a look at this on this weekend if it suits.